app/views/LoginView.js

```diff
@@ -118,8 +118,9 @@ class LoginView extends React.Component {
 
 		const { user, password } = this.state;
 		const { loginRequest } = this.props;
+		const loginEmailPassword = true;
 		Keyboard.dismiss();
-		loginRequest({ user, password });
+		loginRequest({ user, password, loginEmailPassword });
```

Comment:
If you pass these info inside `loginWithPasswordCall`, these changes won't be necessary.